github/ci/prow-deploy/kustom/overlays/kubevirt-prow-control-plane/patches/JsonRFC6902/crier_deployment.yaml

```diff
@@ -26,6 +26,12 @@
 - op: remove
   path: /spec/template/spec/volumes/8
 
+  # removes kubeconfig-k8s-infra-kops-prow-build mount and volume
```

Comment:
IIRC the removal operations must be done from last to first, please see ll. 17-25 and sort the removal accordingly.

Note that you also need into account moving the other indices where entries have been added so that the right ones are removed. (Feeling sorry for you already :-/ )